css/style.css

```diff
@@ -41,10 +103,171 @@ body {
   margin: auto;
 }
 
-header {}
+header {
+  background-color: #033048;
+  color: white;
+}
 
-footer {}
+footer {
+  background-color: #121212;
+  padding: 20px;
+  border-left:2px solid  #f0efef;
+  border-right: 2px solid  #f0efef;
+  border-bottom: 2px solid  #f0efef;}
 
 /******************************************
 /* ADDITIONAL STYLES
-/*******************************************/
\ No newline at end of file
+/*******************************************/
+
+/* Header */
+.box {
+  color:#f9e42e;
+  font-size: 25px;
+  font-weight: 600px;
+  background-color: #033048;
+  border: 1px solid #033048;
+  padding: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+
+.Navigation {
+  background-color: #033048;
+  display: flex;
+  border: solid 3px #033048; 
+  max-width: 100%;
+  justify-content: space-between;
+}
+
+.primary-nav {
+  display: flex;
+  justify-content: space-between;
+  border: solid 3px #033048; 
+}
+
+.nav-button {
+  background-color: pink;
+  border: 1px solid tomato;
+  padding: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.strong:first-child {
+  color: white;
+  font-size: 25px;
+  font-weight: 600px;
+}
+
+/* Body */
+
+.flex {
+  display: flex;
+}
+
+.body {
+  min-height: 100px;
+  min-width: 68px;
+  padding: 10px;
+  background-color: #f0efef;
+  border: solid 3px #f0efef; 
+}
+
+.flex .body:first-child{
+  width: 20px;
+  padding-left: 20px;
+ }
+
+ .flex .body:nth-child(2) {
+  width: 700px;
```

Comment:
instead of setting fixed widths, i'd recommend using percentages here e.g. width: 70%